### DISCUSSION TITLE
[BUG] Adding new repositories to the Favorites section on the "Home" tab doesn‘t save them

### DISCUSSION THREAD

**Discussion by Ein-Tim** [community/community#34782] [Mobile]
When I add new repositories to the favorites list on the "Home" tab  of GitHub mobile for iOS, these changes aren't saved and disappear as soon as I close and restart the app.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.82.0 (79366424)
Device: iPhone Xʀ (iOS 16.0.2)
</details>

**Comment by HenSquared** [comment]
Hi @Ein-Tim thanks for opening this issue. Unfortunately, I'm not able to reproduce this issue. Does this happen with all repositories? Or a specific repository? 

Thanks!

**Reply by Ein-Tim** [reply]
@HenSquared I'm adding a screen recording here, maybe this helps! Can I somehow activate logging in the GitHub app?

https://user-images.githubusercontent.com/67682506/193864275-529a738f-6d31-46dd-9446-6997ee7fb442.mp4

This happens with all repositories.

**Reply by jacortinas** [reply]
👋 Hello, I'm also a dev on the mobile team.

@HenSquared @Ein-Tim What I think is happening here is that there is a limit of 25 favorites. We can definitely look into making this limit larger, but that has implications load time and possible pagination on the home screen. I think at the very least we need to make sure that this limit is expressed to people using the app so it's not a mystery.

**Reply by HenSquared** [reply]
@jacortinas thanks for the investigation on this one. I will create an issue to track our work to surface this error to the user. 

And thanks to @Ein-Tim for the repro video!

**Reply by Ein-Tim** [reply]
Hey @jacortinas 👋 

I managed to get it working by first deleting the 3 repos (as I did in the video), then close the GitHub app and then add the other repos separately. So I don't think a limit is the problem, as now everything works as it should (and I have all 22 repos in the favorites section).
